OPHD/States/MapViewStateEvent.cpp

```diff
@@ -111,9 +111,10 @@ void MapViewState::deployColonistLander()
  */
 void MapViewState::deployCargoLander()
 {
-	auto cc = static_cast<CommandCenter*>(mTileMap->getTile(ccLocation(), 0).structure());
-	cc->foodLevel(cc->foodLevel() + 125);
-	cc->storage() += StorableResources{ 25, 25, 15, 15 };
+	Structure* structure = mTileMap->getTile(ccLocation(), 0).structure();
+	auto& foodProduction = getComponent<FoodProduction>(structure);
+	foodProduction.foodLevel(foodProduction.foodLevel() + 125);
+	structure->storage() += StorableResources{ 25, 25, 15, 15 };
```

Comment:
This is just a mechanical translation, but in general I feel that the deployXxLander functionality could be made more elegant (later, perhaps when converting the landers to components).

Reply:
It could be -- the deploy functionality was sort of hacked together to 'just get it to work' so I could move on to other things.

This code looks a lot cleaner too, I like this.